tests/storage/lvm_test.py

```diff
@@ -1023,6 +1023,63 @@ def test_vg_create_multiple_devices(tmp_storage):
         assert pv.vg_name == ""
 
 
+@requires_root
+@pytest.mark.root
+def test_vg_remove_by_uuid(tmp_storage):
+    dev_size = 10 * GiB
+
+    dev1 = tmp_storage.create_device(dev_size)
+    dev2 = tmp_storage.create_device(dev_size)
+    vg1_name = str(uuid.uuid4())
+    vg2_name = str(uuid.uuid4())
+
+    lvm.createVG(vg1_name, [dev1], "initial-tag", 128)
+    lvm.createVG(vg2_name, [dev2], "initial-tag", 128)
+
+    clear_stats()
+    vg = lvm.getVG(vg1_name)
+    check_stats(hits=0, misses=1)
+
+    lvm.removeVGbyUUID(vg.uuid)
+    check_stats(hits=0, misses=2)
+    # Ensure we have removed the matching VG.
+    with pytest.raises(se.VolumeGroupDoesNotExist):
+        lvm.getVGbyUUID(vg.uuid)
+        lvm.getVG(vg1_name)
+        check_stats(hits=0, misses=4)
```

Comment:
lines 1048,1049 not reachable since we raie in line 1047.
We need to have pytest.raises() block for each check.
We can remove the stats check, so we will have:

    with pytest.raises(se.VolumeGroupDoesNotExist):
        lvm.getVGbyUUID(vg.uuid)

    with pytest.raises(se.VolumeGroupDoesNotExist):
        lvm.getVG(vg.name)

Reply:
Yeah I wasn't sure, but it crossed my mind this may happen. Thanks! I'll fix this.